osu.Game.Tournament/Screens/MapPool/MapPoolScreen.cs

```diff
@@ -245,12 +246,15 @@ private void updateDisplay()
 
             if (CurrentMatch.Value.Round.Value != null)
             {
-                FillFlowContainer<TournamentBeatmapPanel> currentFlow = null;
-                string currentMods = null;
+                FillFlowContainer<TournamentBeatmapPanel>? currentFlow = null;
+                string? currentMods = null;
                 int flowCount = 0;
 
                 foreach (var b in CurrentMatch.Value.Round.Value.Beatmaps)
                 {
+                    if (b.Beatmap == null)
+                        continue;
```

Comment:
why?

Reply:
because `b` and `b.Beatmap` may be null?